addon-test-support/@ember/test-helpers/teardown-context.ts

```diff
@@ -28,28 +29,14 @@ export default function teardownContext(
   if (options !== undefined && 'waitForSettled' in options) {
     waitForSettled = options.waitForSettled!;
   }
+
   return nextTickPromise()
     .then(() => {
-      let { owner } = context;
-
-      _teardownAJAXHooks();
-
-      run(owner, 'destroy');
-      (Ember as any).testing = false;
-
-      unsetContext();
-
-      if (waitForSettled) {
-        return settled();
-      }
-
-      return nextTickPromise();
+      destroy(context);
```

Comment:
Were we only waiting for `settled` before because we were using runloop APIs?

Reply:
If you look just below, we are still `return await settled()` in the finally. I honestly do not know why we were doing it twice.